RobustGenericAttribution/README.md

```diff
@@ -0,0 +1,38 @@
+# Documentation
+
+The **RGA** (Robust Generic Attribution) standard is intended to be a flexible, open, and readable way <!--Insert more marketing bull that sounds good here!--> to define various metadata such as licensing and attribution for an arbitrary collection of files of many types such as sound effects or prototypes. An RGA file contains metadata for all of the files in the same directory as the RGA file (not including subdirectories). The entries in an RGA file contain specific metadata such as the author of the file(s) or a description of any modifications made to them (in compliance with many Creative Commons licenses).
+
+An RGA is a YAML file with the name `attributions.yml`, and contains an arbitrary number of entries as defined below.
+
+Typically whenever new files are being added, a new entry will be added as it likely won't contain the same metadata as existing entries. However, one may append files to an existing entry if the metadata is otherwise identical.
+
+## YAML
+
+An RGA file must be named `attributions.yml`. All values within entries are wrapped in double-quotes (`""`).
+
+The YAML contains an arbitrary number of entries, covering all files in the same directory as the RGA file. An entry is defined as follows:
+
+Key | Meaning
+--- | -------
+`files` | An array of filenames (with extensions) that this entry applies to. The filename order is arbitrary.
```

Comment:
We should support [globs](https://en.wikipedia.org/wiki/Glob_(programming)) here, specially wildcards.

Reply:
```suggestion
`files` | An array of filenames (with extensions) that this entry applies to. The filename order is arbitrary. The `*` wildcard glob is supported (i.e. `*.ogg` denotes all OGG files in the directory).
```